pkgs/applications/virtualization/virtualbox/guest-additions/no-legacy-xorg.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/src/VBox/Additions/linux/Makefile.kmk b/src/VBox/Additions/linux/Makefile.kmk
```

Comment:
What's the source of this patch? If it's self-developed, a short explanation of why we need it would also be nice for people that are not deep in the weeds of vbox guest drivers.

Reply:
Valid point. Will add it. For reference within this PR: https://www.virtualbox.org/changeset/100258/vbox